cmd/eksctl-anywhere/cmd/root.go

```diff
@@ -38,24 +42,55 @@ func init() {
 }
 
 func rootPersistentPreRun(cmd *cobra.Command, args []string) {
-	if err := initLogger(); err != nil {
+	stdout := logger.NewPausableWriter(os.Stdout)
+
+	if err := initLogger(stdout); err != nil {
 		log.Fatal(err)
 	}
+
+	signals.On(func() {
```

Comment:
This isn't really relevant for commands beyond create delete and upgrade so we can probably move to those locations.